client/app/application.coffee

```diff
@@ -2,7 +2,37 @@ module.exports =
 
     listenTo: Backbone.Model.prototype.listenTo
 
-    initialize: ->
+
+    # Initialize a custom error handler for the global app
+    initializeErrorHandler: (window) ->
+        existingDefaultHandler = window.onerror
+
+        applicationErrorHandler = (msg, url, line, col, error) ->
+            # Handler for asynchronous errors
+            @onEventSharingError = (error) ->
+                # TODO find a better way to format a string like this
+                alert [ t('Event sharing failed for event'),
+                        error.event.get('description'),
+                        '(' + t(error.message) + ')' ].join ' '
+
+                return true
+
+            # error = event.error
+            errorHandlerName = 'on' + error.name
```

Comment:
@frankrousseau 
Yes, my problem here is that I want to format the string with the event's name, i.e. :
* "Event sharing failed for event 'birthay party'" in english
* "L'évènement 'birthday party' n'a pu être partagé" in french
Does Coffeescript string concatenation and Transifex translation work well together ? If yes, I will use the coffee way.